kstreams/streams.py

```diff
@@ -103,6 +103,7 @@ def __init__(
         self.name = name or str(uuid.uuid4())
         self.deserializer = deserializer
         self.running = False
+        self.is_processing = asyncio.Condition()
```

Comment:
This could be an `asyncio.Lock()` as the event part of it is not used

Reply:
yes!